### PR TITLE
Additions to the observation unit UI 

### PR DESCRIPTION
**Pull request by stuzart**
Consolidated earlier changes with those done by PhenomUK.
Added ability to associate a data file or sample

* Fix for #1961

### CODE REVIEW

File: app/views/observation_units/new.html.erb
Comment (AndrewWood94) [inline app/views/observation_units/new.html.erb:1]: Text for no Investigation/Study available is still referencing Assay no Obs Unit

**Review by AndrewWood94 — CHANGES_REQUESTED** [review]
Minor changes needed on new Obs Unit file, otherwise all good

File: app/views/observation_units/new.html.erb
Comment (stuzart) [reply app/views/observation_units/new.html.erb:1]: good spot

**Comment by stuzart** [comment]
> Minor changes needed on new Obs Unit file, otherwise all good

Thanks. I fixed the error and based on this comment gone straight ahead and merged

**Pull request merged**